.travis.yml

```diff
@@ -95,8 +95,6 @@ matrix:
       jdk: oraclejdk9
     - env: PHASE='-Prake -Dtask=test:mri:jit'
       jdk: oraclejdk9
-    - env: PHASE='-Prake -Dtask=spec:ruby:fast'
```

Comment:
:scissors: necessary?

Reply:
Yeah I didn't noticed this was for JDK9, I removed the commit.